### DISCUSSION TITLE
ANTs PET Motion Correction

### DISCUSSION THREAD

**Discussion by noahg-neuroimage** [ANTsX/ANTs#1637] [Q&A]
I'm a beginner ANTs user, interested in using motion correction tools for PET data. I've successfully performed some motion corrections based on the example antsMotionCorr module, and I'd like to evaluate the performance of chosen parameters. Does ANTs provide an estimated motion error, or is this something I can calculate based on the output?

**Comment by ntustison** [comment]
Our ANTsR and ANTsPy wrappers for ``antsMotionCorr`` have various measures.   See [here](https://gist.github.com/ntustison/12a656a5fc2f6f9c4494c88dc09c5621#motion-correction) for an example.
